test/blackbox-tests/test-cases/dune-cache/size.t/run.t

```diff
@@ -0,0 +1,41 @@
+Testing the dune cache size command.
+
+  $ export DUNE_CACHE=enabled
+  $ export DUNE_CACHE_ROOT=$PWD/.cache
+
+  $ cat > config <<EOF
+  > (lang dune 3.7)
+  > (cache enabled)
+  > (cache-storage-mode copy)
+  > EOF
+
+
+  $ cat > dune-project <<EOF
+  > (lang dune 3.7)
+  > EOF
+  $ cat > dune <<EOF
+  > (rule
+  >  (targets target_a)
+  >  (action
+  >   (ignore-outputs
+  >    (run dd if=/dev/zero of=target_a bs=1k count=1))))
+  > (rule
+  >  (deps target_a)
+  >  (targets target_b)
+  >  (action
+  >   (with-outputs-to
+  >    target_b
+  >    (run dune_cmd stat size target_a))))
```

Comment:
Why is this action in the test? This test seems pretty complex for no good reason. (But if it needs to be here for some reason, please clarify in a comment.)